.github/workflows/publish.yml

```diff
@@ -84,43 +80,38 @@ jobs:
         id: cache-modules
         uses: actions/cache@v1
         with:
-          path: chat/node_modules
-          key: ${{ runner.OS }}-build-${{ hashFiles('chat/package.json') }}
+          path: node_modules
+          key: ${{ runner.OS }}-build-${{ hashFiles('package.json') }}
       - name: Install node modules
-        working-directory: chat
         if: steps.cache-modules.outputs.cache-hit != 'true'
         run: yarn install --frozen-lockfile
       - name: Build project into nodejs app
-        working-directory: chat
         run: yarn build
       - name: Compile project into nodes bin
-        working-directory: chat
         run: yarn compile
       - name: Zip node binary for uploading
-        working-directory: chat
         run: yarn zip:${{ matrix.displayName }}
       - name: Upload binaries to GitHub workflow artifacts store
-        uses: actions/upload-artifact@v2-preview
+        uses: actions/upload-artifact@v2
         with:
           name: hopr-chat-nodebin-${{ matrix.displayName }}.zip
-          path: chat/hopr-chat-nodebin-${{ matrix.displayName }}.zip
+          path: hopr-chat-nodebin-${{ matrix.displayName }}.zip
 
   publish_chat_dist:
     name: Creates release with node binary per OS
-    if: startsWith(github.ref, 'refs/heads/releases/chat')
     runs-on: ubuntu-latest
     needs: [build_chat_binaries]
     steps:
       - name: Get HOPR Chat (linux)
-        uses: actions/download-artifact@master
+        uses: actions/download-artifact@v2
```

Comment:
Why not master here?

Reply:
`master` changes, whereas `v2` uses a fixed tag we know it's working.

Reply:
oh, I misread - this is the action version, I see.